### ISSUE TITLE
Browser support: browserify vs other bundlers

### ISSUE THREAD

**Issue by michaelstack** [debug-js/debug#903]
The readme has Browser Support section. That section gives instructions for working with browserify. It does not make any mention of other bundlers.

I have tried adding debug to a project using webpack. It appeared to work just fine.

Is there some reason why the readme only gives instructions for browserify? That is, are there any known issues with other bundlers that do not happen with browserify?

**Comment by Qix-** [comment]
Yes, the readme is horribly outdated and bloated 🙃 It's been on my list for years to rewrite it, and it'll happen for sure for the next major release.

**Comment by michaelstack** [comment]
@Qix- thanks for the quick response.
